src/actions/teacherBoard.js

```diff
@@ -0,0 +1,177 @@
+import { toastr } from 'react-redux-toastr';
```

Comment:
We could actually rename the `TeacherBoard` into `Classrooms`, right? Or are we going to include something else inside?